tests/console/btrfsmaintenance.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check btrfsmaintenance for functionality
+# - run btrfs-balance.sh and btrfs-scrub.sh
+# - Check if btrfsmaintenance-refresh.service is present and started properly
+# - Check if btrfs-scrub is scheduled
+# - Check if btrfs-balance is scheduled
+# - Checks for btrfs-related cron jobs after uninstalling btrfsmaintenance
+# Maintainer: Felix Niederwanger <felix.niederwanger@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
```

Comment:
this is not needed

Reply:
also version_utils ?

Reply:
Removed the unnecessary use packages and re-scheduled the runs.